src/content/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks.mdx

```diff
@@ -64,6 +64,7 @@ The Ruby agent provides robust instrumentation for these web frameworks.
 | **Grape** | 1.7.x, 1.6.x, 1.5.x, 1.4.x, 1.3.x, 1.2.x | Grape 0.2.0: Last fully supported by agent v8.16.0. Older versions may not be fully supported. |
 | **Padrino** | 0.15.x | Older versions (e.g., 0.14.x) last fully supported by agent v8.16.0. |
 | **Roda** | 3.x |  |
+| **View Component** | 2.53.x and above |  |
```

Comment:
I think this might make more sense in the "Other" section, it's a framework for view objects, but not quite a web framework like the others in this table